main/modes/quickSettings/quickSettings.c

```diff
@@ -68,23 +80,19 @@ static int32_t quickSettingsMenuFlipItem(const char* label);
 
 static const char quickSettingsName[] = "Settings";
 
-static const char quickSettingsLeds[] = "LEDs ";
-static const char quickSettingsSfx[]  = "Sound Effects ";
-static const char quickSettingsBgm[]  = "Music ";
-
-static const char quickSettingsOn[]    = "On";
-static const char quickSettingsOff[]   = "Off";
-static const char quickSettingsMuted[] = "Muted";
+static const char quickSettingsLeds[] = "LED Brightness";
+static const char quickSettingsSfx[]  = "SFX Volume";
+static const char quickSettingsBgm[]  = "Music Volume";
+static const char quickSettingsBacklight[] = "Screen Brightness: ";
 
-static const char* const quickSettingsOptionsLeds[] = {
-    quickSettingsOff,
-    quickSettingsOn,
-};
-
-static const char* const quickSettingsOptionsAudio[] = {
-    quickSettingsMuted,
-    quickSettingsOn,
-};
+static const char quickSettingsLedsOff[] = "LEDs Off";
+static const char quickSettingsLedsMax[] = "LED Brightness: Max";
+static const char quickSettingsSfxMuted[] = "SFX Muted";
+static const char quickSettingsSfxMax[]= "SFX Volume: Max";
+static const char quickSettingsBgmMuted[] = "Music Muted";
+static const char quickSettingsBgmMax[]= "Music Volume: Max";
+static const char quickSettingsBacklightOff[] = "Screen Backlight Off";
+static const char quickSettingsBacklightMax[] = "Screen Brightness: Max";
```

Comment:
It would be real nice to have consistent, shared strings between main menu settings and quick settings.

Reply:
Ah, yeah... I went for longer strings here since the main menu's settings are very abbreviated and there's not a ton of room for longer names with the logbook font. I'll see what I can come up with that's consistent and fits, maybe I can use the logbook font here too then.

Reply:
OK, you thought this through already. Smaller font does mean more space to work with, so that's fine. You don't need to bring them in line.